src/main/java/com/synopsys/integration/detect/lifecycle/run/step/ContainerScanStepRunner.java

```diff
@@ -20,21 +22,23 @@ public class ContainerScanStepRunner {
 
     private final OperationRunner operationRunner;
     private UUID scanId;
+    private final Gson gson;
```

Comment:
A descriptive name can be chosen.

Reply:
This is a generic gson variable being passed from Rapid/Intelligent Mode Step Runner classes (which receive it from the `DetectRun` class so not specific to container scanning) and is being called `gson` in other step runners too, hence continuing the same naming convention here.